src/var/timeoutSet.js

```diff
@@ -0,0 +1,4 @@
+define(function() {
+	/* global setTimeout: false */
+	return window.setTimeout || setTimeout;
```

Comment:
Oh, then it would duplicate it everywhere :-(, would be really awkward though, or maybe just link to this pull would be enough?